cli/src/main/java/org/jboss/sbomer/cli/feature/sbom/command/GradleCycloneDxGenerateCommand.java

```diff
@@ -51,7 +51,13 @@ protected Path doGenerate(String buildCmdOptions) {
         configureProcessEnvironmentVariable(buildCmdOptions, processBuilder);
         configureProcessMainBuildCommands(buildCmdOptions, processBuilder);
 
+        processBuilder.environment()
+                .put(
+                        "JAVA_OPTS",
+                        "-Dorg.gradle.jvmargs=-XX:InitialRAMPercentage=60.0 -XX:MaxRAMPercentage=60.0 -XX:+ExitOnOutOfMemoryError -XshowSettings:vm");
+
         processBuilder.command().add("cyclonedxBom");
+        processBuilder.command().add("--no-daemon");
```

Comment:
@vibe13 I would like to point you to this specifically. To not start another JVM, I would probably run the generation in the "client" VM and adjust the settings as above. Do you see any drawbacks?